src/supported_apps.ts

```diff
@@ -277,4 +277,16 @@ export const supportedApps: SubstrateAppParams[] = [
     slip0044: 0x80000219,
     ss58_addr_type: 126,
   },
+  {
+    name: "Enjin",
+    cla: 0xb9,
+    slip0044: 0x80000483,
+    ss58_addr_type: 2135,
```

Comment:
I don't see this ss58 prefix reserved [here](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json).

Reply:
Here it is: https://github.com/paritytech/ss58-registry/pull/187